app/backend/prisma/schema.prisma

```diff
@@ -134,51 +113,54 @@ model VerificationToken {
 }
 
 model Team {
-  id         String    @id @default(cuid())
-  title      String
-  slug       String    @unique
-  theme      Theme     @default(BLUE)
-  inviteKey  String    @unique @default(cuid())
-  createdAt  DateTime  @default(now())
-  updatedAt  DateTime  @updatedAt
-  archivedAt DateTime?
-
-  teamMemberships TeamMembership[]
+  id              String           @id @default(cuid())
+  title           String
+  slug            String           @unique
+  theme           Theme            @default(BLUE)
+  inviteKey       String           @unique @default(cuid())
+  createdAt       DateTime         @default(now())
+  updatedAt       DateTime         @updatedAt
+  archivedAt      DateTime?
   customers       Customer[]
   projects        Project[]
-}
-
-enum Theme {
-  GRAY
-  RED
-  YELLOW
-  GREEN
-  BLUE
-  INDIGO
-  PURPLE
-  PINK
+  teamMemberships TeamMembership[]
 }
 
 model TeamMembership {
-  id          String   @id @default(cuid())
-  memberSince DateTime @default(now())
-  userId      String
-  teamId      String
-  role        Role     @default(MEMBER)
-
-  user User @relation(fields: [userId], references: [id], onDelete: Cascade)
-  team Team @relation(fields: [teamId], references: [id], onDelete: Cascade)
+  id              String   @id @default(cuid())
+  memberSince     DateTime @default(now())
+  userId          String
+  teamId          String
+  role            Role     @default(MEMBER)
+  capacityMinutes Int?
```

Comment:
availableMinutesPerWeek

please rename, sorry - but the old name is not clear enough - my fault.
Please rename also in the graphQL and others...